lib/clientsidescripts.js

```diff
@@ -47,24 +47,46 @@ function wrapWithHelpers(fun) {
  *     be passed as a parameter.
  */
 functions.waitForAngular = function(rootSelector, callback) {
-  var el = document.querySelector(rootSelector);
-
-  try {
-    if (!window.angular) {
-      throw new Error('angular could not be found on the window');
+  if (!rootSelector) {
```

Comment:
Yup, the root selector gets defaulted to 'body' in the constructor for a `Protractor` instance. How would someone set `useAllAngularAppRoots` to true accidentally?